packages/react-native-reanimated/apple/reanimated/apple/REAModule.mm

```diff
@@ -293,11 +291,12 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body
   auto jsCallInvoker = self.bridge.jsCallInvoker;
 #endif // defined(RCT_NEW_ARCH_ENABLED) && REACT_NATIVE_MINOR_VERSION >= 75
   auto jsiRuntime = reinterpret_cast<facebook::jsi::Runtime *>(self.bridge.runtime);
+  auto isBridgeless = ![self.bridge isKindOfClass:[RCTCxxBridge class]];
```

Comment:
I tested it on `0.74, 0.75, 0.76` and it worked well there. Moreover, it's only used to inject `isBridgeless` into the React Runtime. If we want, we can drop it altogether in some follow-up PRs.

Reply:
I think we can safely assume that bridgeless mode is always enabled on Fabric. Let's just have `auto isBridgeless = true` for now, what do you think?

Reply:
We still support 0.74 though, where Bridgeful could be a case.

Reply:
Right